website/docs/getting-started/configuration/glossary.md

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 5
+---
+
+# Glossary - Environment Variables
+
+## Using environment variables
+
+Hutch can be configured using the following source in [the usual .NET way](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration), in order of precedence:
+-  `appsettings.json` adjacent to the binary (`HutchAgent.dll`)
+- Environment Variables (with double underscore `__` as a hierarchical separator)
+- Command line arguments
+- (.NET User Secrets in development)
+
+## Available values
+
+The following table shows the environment variables that can used in a production environment as well as a development environment. It also gives a brief description along with the variable's type. The variables will default to the value in the "Default" column if one is specified. Variables with no default value must be provided yourself. Variables marked with **(Required)** must be given or Hutch will not work.
+
+NB: `StoreDefaults__Host`, unlike the other `StoreDefaults__*` variables is not required if you have a S3 store running on your local machine at port 9000. If you are using a remote S3 instance or a different port, you must set this variable.
+
+| Varibale name | Description | Type | Default |
+|---------------|-------------|------|---------|
+|`Paths__WorkingDirectoryBase`|Hutch's working directory|`string`|`$HOME/hutch-workdir`|
```

Comment:
Isn't this like the third time you've described the same options? Have one source of truth and refer to it, otherwise this is gonna become unmaintainable fast.